### PR TITLE
S02us01 02 layaout banner

### LINKED ISSUE #19: US01-02: Création d'une bannière modifiable et appelable sur différente pages
créer un layout pour la bannière

### PR DESCRIPTION
**Pull request by bouboumael**
Create a banner to include it in all pages and be able to modify the title and image

### CODE REVIEW

**Review by ChristianOlmedo — COMMENTED** [review]
_bannerExemple.html.twig you have use to much tabs line 4

**Review by ChristianOlmedo — DISMISSED** [review]

File: src/View/Includes/Banner/layoutBanner.html.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="container-fluid p-0 banner">
+	{% block picture %}{% endblock %}
+	{% block pageTitle %}{% endblock %}
+</div>
```
Comment (LucyMerlier) [inline src/View/Includes/Banner/layoutBanner.html.twig]: a layout for components seems overkill?

File: src/View/Includes/Banner/pages/_bannerExemple.html.twig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% extends "/Includes/Banner/layoutBanner.html.twig" %}
+
+{% block picture %}
+    <img class="img-banner" src="/assets/images/collector.webp" alt="mascotte">
+{% endblock %}
+{% block pageTitle %}
+    <h1 class="text-light">Haras du Val de Loire</h1>
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/Includes/Banner/pages/_bannerExemple.html.twig]: this extends a component? +these blocks seem

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Includes/Banner/layoutBanner.html.twig (hunk 1 above)
Comment (bouboumael) [reply src/View/Includes/Banner/layoutBanner.html.twig]: yes it's true. correction done

File: src/View/Includes/Banner/pages/_bannerExemple.html.twig (hunk 2 above)
Comment (bouboumael) [reply src/View/Includes/Banner/pages/_bannerExemple.html.twig]: correction done

**Review by NEL45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**